muted-tests.yml

```diff
@@ -97,9 +97,6 @@ tests:
 - class: "org.elasticsearch.xpack.searchablesnapshots.FrozenSearchableSnapshotsIntegTests"
   issue: "https://github.com/elastic/elasticsearch/issues/110408"
   method: "testCreateAndRestorePartialSearchableSnapshot"
-- class: org.elasticsearch.test.rest.yaml.CcsCommonYamlTestSuiteIT
-  method: test {p0=search.vectors/41_knn_search_half_byte_quantized/Test create, merge, and search cosine}
```

Comment:
In the test we use an index with  `number_of_shards: 1`; do CCS tests override that / create different settings?

Reply:
@pmpailis CCS means more than one cluster. So by definition, even if an index has only one shard, the docs could be split across different indices on different clusters.

Reply:
Yeah, the number of shards is still `1` for all the indices over all the clusters, its just now spread over more than one cluster :)